### PR TITLE
Magiclysm - New Spells 1

### PR DESCRIPTION
**Pull request by foulman**
Fresh spells for the masses!
Biomancer, Stormshaper, Druid, and Earthshaper. 
Wish I could take credit, but I can't.

#### Summary
```SUMMARY: Mods "Magiclysm - New Spells 1"```

#### Purpose of change
Not much to say, new spells.

#### Describe the solution
making spells of course.

#### Describe alternatives you've considered
Not making spells. A terrible alternative.

#### Testing
They have been playtested.

### CODE REVIEW

**Comment by LaVeyanFiend** [comment]
For futureproofing:

For the damaging spells, what does it say if a monster casts it on *you*? 👀

File: data/mods/Magiclysm/Spells/stormshaper.json [hunk 1]
```diff
@@ -205,5 +205,108 @@
...
+  },
+  {
+    "id": "stormshaper_ionization",
+    "type": "SPELL",
+    "name": "Ionization",
+    "description": "By manipulating the charge in the air, you can conjure a sharp snap of lightning over a wide area.  While its destructive potential is a far cry from natural lightning, the light and thunderclap produced will leave your foes reeling.",
+    "valid_targets": [ "hostile", "ground", "self", "ally" ],
+    "effect": "target_attack",
+    "extra_effects": [ { "id": "stormshaper_ionization_thunderclap" } ],
+    "flags": [ "SOMATIC", "NO_LEGS" ],
```
[the remark this answers is not in the document]
Comment (foulman) [reply data/mods/Magiclysm/Spells/stormshaper.json]: thunderclap makes the sound of a flashbang, this spell does no damage to cause LOUDness

File: data/mods/Magiclysm/Spells/stormshaper.json [hunk 2]
```diff
@@ -205,5 +205,108 @@
...
+    "max_level": 30,
+    "min_range": 12,
+    "max_range": 24,
+    "range_increment": 0.4
+  },
+  {
+    "id": "stormshaper_summon_wisps",
+    "type": "SPELL",
+    "name": "Ignus Fatuus",
+    "description": "Summons ghostly foxfire worked from living marsh vapor, to lead your enemies astray.  With more experience, this spell can conjure multiple ghost lights.",
```
[the remark this answers is not in the document]
Comment (foulman) [reply data/mods/Magiclysm/Spells/stormshaper.json]: Stormshapers also have the element of air, marsh vapors, wisps are swamp/air creatures.

**Comment by foulman** [comment]
> 
> 
> For futureproofing:
> 
> For the damaging spells, what does it say if a monster casts it on _you_? 👀

That goes in the monster attack entry iirc.

File: data/mods/Magiclysm/Spells/animist.json [hunk 3]
```diff
@@ -186,5 +186,32 @@
     "spell_class": "ANIMIST",
     "energy_source": "MANA",
     "flags": [ "PERMANENT", "NO_LEGS", "CONCENTRATE" ]
+  },
+  {
+    "id": "stormshaper_summon_wisps",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/animist.json]:
```suggestion
    "id": "summon_wisps",
```
or animist, but i don't particularly have a strong opinion about the prefixes matching the classes

**Comment by KorGgenT** [comment]
so i just now noticed these don't have scrolls. i'm going to go ahead and test out the spells in game. reading them they seem fine. this should be able to make it in despite content freeze and when we wrap it up i'll make the skill pr

**Comment by KorGgenT** [comment]
yeah the spells seem fine. they just need at minimum a scroll for each one that gets assigned to one itemgroup that has a spell tier

**Pull request merged**